src/Microsoft.AspNet.Mvc.Core/Rendering/Html/DefaultHtmlGenerator.cs

```diff
@@ -25,7 +26,7 @@ public class DefaultHtmlGenerator : IHtmlGenerator
             typeof(DefaultHtmlGenerator).GetTypeInfo().GetDeclaredMethod(nameof(ConvertEnumFromString));
 
         private readonly AntiForgery _antiForgery;
-        private readonly IScopedInstance<ActionBindingContext> _bindingContextAccessor;
+        private readonly IClientModelValidatorProvider _clientModelValidatorProvider;
```

Comment:
Do we care about allowing `IClientModelValidators` to be changed via `ResourceFilters` (similar to your `IOutputFormatter` change)?